stretch_core/launch/d435i_basic.launch.py

```diff
@@ -6,17 +6,17 @@
 from launch_ros.substitutions import FindPackageShare
 
 # https://github.com/intel-ros/realsense
-launch_arguments = {'accel_fps': '63',
-                    'gyro_fps': '200',
-                    'depth_fps': '15',
+launch_arguments = {'accel_fps': '63.0',
+                    'gyro_fps': '200.0',
+                    'depth_fps': '15.0',
```

Comment:
ROS2 is very picky about parameter types. All the fps parameters are doubles.

Reply:
Yeah they are nice catch.